v2/tools/config/go-templates/atlas_client.mustache

```diff
@@ -0,0 +1,53 @@
+package {{packageName}} // import "go.mongodb.org/atlas/mongodbatlas/v2"
+
+import (
+	"fmt"
+	"net/http"
+	"runtime"
+
+	"github.com/mongodb-forks/digest"
+)
+
+const (
+	// CloudURL is default base URL for the services.
+	DefaultCloudURL = "https://cloud.mongodb.com/"
+	// Version the version of the current API client. Should be set to the next version planned to be released.
+	Version    = "2.0.0-dev1"
+	ClientName = "go-mongodbatlas-versioned"
```

Comment:
[q, not blocking] Is the final plan to leverage a variable here to generate the version info for the client?

Reply:
Yes. Actually, that is the main reason why the client was moved to the mustache template :)